examples/tutorial/jupyter/execution/pandas_on_dask/cluster/exercise_5.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Often in practice we have a need to exceed the capabilities of a single machine. Modin works and performs well \n",
+    "in both local mode and in a cluster environment. The key advantage of Modin is that your python code does not \n",
+    "change between local development and cluster execution. Users are not required to think about how many workers \n",
+    "exist or how to distribute and partition their data; Modin handles all of this seamlessly and transparently.\n",
+    "\n",
+    "![Cluster](../../../img/modin_cluster.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Extra requirements for AWS authentication\n",
+    "\n",
+    "First of all, install the necessary dependencies in your environment:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install dask_cloudprovider[aws]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step is to setup your AWS credentials. One can set ``AWS_ACCESS_KEY_ID``, ``AWS_SECRET_ACCESS_KEY``\n",
+    "and ``AWS_SESSION_TOKEN`` (Optional) (refer to [AWS CLI environment variables](\"https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html\") to get more insight on this):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"AWS_ACCESS_KEY_ID\"] = \"\"\n",
+    "os.environ[\"AWS_SECRET_ACCESS_KEY\"] = \"\"\n",
+    "os.environ[\"AWS_SESSION_TOKEN\"] = \"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Starting and connecting to the cluster\n",
+    "\n",
+    "This example starts 1 scheduler node (m5.24xlarge) and 6 worker nodes (m5.24xlarge), 576 total CPUs. Please keep in mind, the scheduler node manages cluster operation but doesn't perform any executions.\n",
+    "\n",
+    "You can check the [Amazon EC2 pricing](https://aws.amazon.com/ec2/pricing/on-demand/).\n",
+    "\n",
+    "Dask cluster can be deployed in different ways (please read [Dask documentaion](\"https://docs.dask.org/en/latest/deploying.html\") to get more information about it), but in this tutorial we will use the ``EC2Cluster`` from [dask_cloudprovider](https://cloudprovider.dask.org/en/latest/) to create and initialize a Dask cluster on Amazon Web Service (AWS).\n",
+    "\n",
+    "**Note**: EC2Cluster uses a docker container to run the scheduler and each of the workers. Probably you need to use another docker image depending on your python version and requirements. You can find more docker-images [here](\"https://hub.docker.com/u/daskdev\").\n",
```

Comment:
The link to "here" doesn't work, why?

Reply:
The extra character "" before link, I will fix it, thanks.